prost-build/src/lib.rs

```diff
@@ -942,9 +943,7 @@ impl Config {
     }
 
     /// Configures the code generator to format the output code via `prettyplease`.
-    ///
-    /// By default, this is enabled but if the `format` feature is not enabled this does
-    /// nothing.
+    #[cfg(feature = "format")]
```

Comment:
I would like it to be clear that this function is only available with the `format` feature. It appears that cargo doc will only show an information balloon for modules with a feature config.

```suggestion
    /// 
    /// Enabled by default.
    ///
    /// This configuration option is only available with the `format` feature enabled.
    #[cfg(feature = "format")]
```

Reply:
What do you mean about information balloon?

Here's an example of it being shown for functions too: https://docs.rs/tokio/latest/tokio/net/struct.TcpStream.html#method.try_read_buf

I think that would be the more "correct way". This is unfortunately not enabled by default in cargo doc [yet](https://github.com/rust-lang/rust/issues/43781). One needs to add `#![cfg_attr(docsrs, feature(doc_auto_cfg))]` to each `lib.rs` and the bellow snippet to the following `Cargo.toml`s (maybe possible in workspace).

```toml
[package.metadata.docs.rs]
all-features = true
rustdoc-args = ["--cfg", "docsrs"]
```

I would personally be in favor of doing the latter, and open a separate PR for this as well 🤗

Reply:
Yes, I mean that information balloon. Good that it can be shown for functions as well. Too bad it is not the default. A command is fine for now.